src/helpers/DateConverter.ts

```diff
@@ -0,0 +1,4 @@
+export function epochToUtc (date: number) {
+  const convertedDate = new Date(date * 1000);
+  return convertedDate.toDateString();
+}
```

Comment:
I was using `snake_case` for JS functions and I guess also for filenames because they all start with a lowercase:

- `sanitizer.ts`
- `slug.ts`

Reply:
It would be good to add a unit test for the function but [it's a pain to add unit tests with Nuxt](https://github.com/torrust/torrust-index-gui/issues/164).

Reply:
> Hi @mario-nt since we are using TypeScript maybe we can introduce a new `Timestamp` type to make it explicit what the function needs.

Maybe something like:

```Typescript
type Timestamp = number;

export function epochToUtc (timestamp: Timestamp) {
  new Date(timestamp.toMilliseconds()).toDateString();
}
```

But you can even use a field and check that the number is an integer.